server/README.md

```diff
@@ -0,0 +1,22 @@
+# 🥏 UltimateApp Server
+
+Server to host the drills for ultimate app based on Mongodb, ApolloServer and Graphql.
+
+## ⌨️ Development
+
+You need a recent version of nodejs, and a [running local mongodb server](https://docs.mongodb.com/manual/installation/#mongodb-community-edition-installation-tutorials)
+
+```
+git clone git@github.com:UltimateApp/UltimateApp.git
+cd UltimateApp/server
+npm install
+node index.js
+```
+
+## TODO
+
+- [ ] Use it in the app
```

Comment:
The big question is what for? I don't think we should use it just to use it.

What do we need this for?